benchmarks/list.js

```diff
@@ -162,7 +145,6 @@ var run = bench([
   threeEntries,
   fiveHundredEntries,
   fiveHundredEntriesAndProperties,
-  fiveHundredEntriesAndKnownProperties,
   patrunThreeEntries,
   patrunFiveHundredEntriesAndProperties
 ], 100000)
```

Comment:
What do you mean by deep? The number of properties? Patrun does not offer multiple nested levels of objects. We did, but it was slowing down everything to a crawl (no inlining). Also, they make the software brittle IMHO, so they should not be used in the first place.